diff --git a/cmake/LabSound.cmake b/cmake/LabSound.cmake
index 82b3326e..583b1640 100644
--- a/cmake/LabSound.cmake
+++ b/cmake/LabSound.cmake
@@ -44,9 +44,9 @@ elseif (LABSOUND_USE_RTAUDIO)
     )
 endif()
 
-option(LABSOUND_INTERNAL_LIBSAMPLERATE, "Use internal libsamplerate", ON)
+option(LABSOUND_INTERNAL_LIBSAMPLERATE "Use internal libsamplerate" ON)
 if (LABSOUND_INTERNAL_LIBSAMPLERATE)
-    set(LABSOUND_LSR "${LABSOUND_ROOT}/src/internal/src/samplerate.c")
+    set(LABSOUND_LSR "${LABSOUND_ROOT}/src/internal/src/libSampleRate.c")
 else()
     find_package(libsamplerate)
     if (NOT libsamplerate_FOUND)
