diff --git a/CMakeLists.txt b/CMakeLists.txt
index ddc8716..735e9d0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -68,7 +68,7 @@ set(SOURCES
     src/values.hpp
 )
 
-add_library(lua-rapidjson MODULE ${SOURCES})
+add_library(lua-rapidjson STATIC ${SOURCES})
 if(${CMAKE_VERSION} VERSION_LESS "3.1.0")
     if(CMAKE_COMPILER_IS_GNUCXX)
         execute_process(COMMAND "${CMAKE_CXX_COMPILER} -dumpversion" OUTPUT_VARIABLE GCC_VERSION)
